ts/common/tests/sheet_helpers_test.ts

```diff
@@ -613,6 +614,31 @@ describe('Test emails', function () {
     tearDownStubs(stubs);
   });
 
+  it('sends emails in CSV format', function () {
+    const output = emailAlertBody('Rule', [
+      {
+        value: '1',
+        anomalous: true,
+        fields: { 'Column A': 'A1', 'Column B': 'B1' },
+      },
+      {
+        value: '2',
+        anomalous: true,
+        fields: { 'Column A': 'A2', 'Column B': 'B2' },
+      },
+    ]);
+
+    expect(output).to.eql(
+      `----------
+      Rule:
+      ----------
+
+      Column A,Column B,Value
+      A1,B1,1
```

Comment:
@Gustavorm545 Is CSV format going to work for emails?

Reply:
@smcjones We have made some changes to the sendEmailAlert to have a table format instead of comma separated. Please refer to the attached file in case you want to take a look.
[sendEmailAlert.txt](https://github.com/user-attachments/files/17961604/sendEmailAlert.txt)